megamek/i18n/megamek/common/enums.properties

```diff
@@ -0,0 +1,9 @@
+# This is used to store any common/enums Resources
+## Enums
+# SkillLevel Enum
+SkillLevel.NONE.text=None
+SkillLevel.ULTRA_GREEN.text=Ultra-Green
+SkillLevel.GREEN.text=Green
+SkillLevel.REGULAR.text=Regular
+SkillLevel.VETERAN.text=Veteran
+SkillLevel.ELITE.text=Elite
```

Comment:
Might as well add a Heroic while we're here, I've seen it mentioned in a few of the rulebooks.

Reply:
I forgot to close this one... was going to use it to simplify a few open PRs, but because this never merged I used worse workaround instead.

However, I do want to add a better level than current when I finish, so I will add it when I restart working on skills.